lesson_25/amiyahjones/src/main.tsx

```diff
@@ -0,0 +1,31 @@
+import App from './App.tsx';
+import {Home} from './pages/Home/Home.tsx';
+import {QueryClient, QueryClientProvider} from '@tanstack/react-query';
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import {RouterProvider, createBrowserRouter} from 'react-router-dom';
+
+import './index.scss';
+
+const queryClient = new QueryClient();
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <App />,
+    children: [
+      {
+        path: '/',
+        element: <Home />,
+      },
```

Comment:
You can add the link to the new page by adding a child here:

`
{
    path: '/CreateProgram',
    element: <CreateProgram />,
}
`

Reply:
thank you so much!